.circleci/config.yml

```diff
@@ -24,7 +24,7 @@ apt-run: &apt-install
   name: Install apt packages
   command: |
     sudo apt update
-    sudo apt install -y graphviz build-essential libopenjp2-7 python3-dev
+    sudo apt install -y libopenjp2-7 python3-dev
```

Comment:
We don't need those two installs. "python3-dev" gives us the python 3.7 we need for the figure tests.